drivers/SmartThings/zigbee-switch/src/test/test_aqara_light.lua

```diff
@@ -53,21 +53,24 @@ end
 
 test.set_test_init_function(test_init)
 
+test.register_coroutine_test(
+  "Handle added lifecycle",
+  function()
+    test.socket.zigbee:__set_channel_ordering("relaxed")
+    test.socket.device_lifecycle:__queue_receive({ mock_device.id, "added" })
+
+    test.socket.zigbee:__expect_send({
+      mock_device.id, cluster_base.write_manufacturer_specific_attribute(mock_device, PRIVATE_CLUSTER_ID,
+      PRIVATE_ATTRIBUTE_ID, MFG_CODE, data_types.Uint8, 1) })
+  end
+)
+
 test.register_coroutine_test(
   "Configure should configure all necessary attributes and refresh device",
   function()
     test.socket.device_lifecycle:__queue_receive({ mock_device.id, "doConfigure" })
     test.socket.zigbee:__set_channel_ordering("relaxed")
 
-    test.socket.zigbee:__expect_send(
-      {
-        mock_device.id,
-        cluster_base.write_manufacturer_specific_attribute(mock_device, PRIVATE_CLUSTER_ID, PRIVATE_ATTRIBUTE_ID
-          , MFG_CODE, data_types.Uint8, 1)
-      }
-    )
-    test.socket.zigbee:__expect_send({ mock_device.id, Level.attributes.OnTransitionTime:write(mock_device, 0) })
-    test.socket.zigbee:__expect_send({ mock_device.id, Level.attributes.OffTransitionTime:write(mock_device, 0) })
```

Comment:
with the changes, shouldn't we _expect_ to see these values sent during configure for this device?

Reply:
Acutually this is one thing I want to ask about the unit TC.

The code has changed to use function of `preferences.lua` with expecting same functionality for the Dimmer Controller T1 Pro. 
```diff
-  device:send(Level.attributes.OnTransitionTime:write(device, 0))
-  device:send(Level.attributes.OffTransitionTime:write(device, 0))
+ preferences.sync_preferences(self, device)
```

When I keep this code at `test_aqara_light.lua`, the test failed. I assume that the lua test doesn't expect message from other files(`preferences.lua`). Could you let me know your idea?
```
    test.socket.zigbee:__expect_send({ mock_device.id, Level.attributes.OnTransitionTime:write(mock_device, 0) })
    test.socket.zigbee:__expect_send({ mock_device.id, Level.attributes.OffTransitionTime:write(mock_device, 0) })
```

Reply:
It's because the mock device's `preference` map is `nil`. `get_profile_definition` doesn't auto-populate it, you have to define it. Example: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/virtual-switch/src/test/test_virtual_switch.lua#L9

Reply:
Thanks you. I updated the PR.